committee-code-of-conduct/election.md

```diff
@@ -76,3 +88,4 @@ votes receiving the longest term.
 
 [Condorcet]: https://en.wikipedia.org/wiki/Condorcet_method
 [Schulze method]: https://en.wikipedia.org/wiki/Schulze_method
+[kubernetes-dev@googlegroups.com]: mailto:kubernetes-dev@googlegroups.com
```

Comment:
tiny nit: can we instead link https://groups.google.com/forum/#!forum/kubernetes-dev so that folks can see the archive/list of emails?